File: 01gpio/05rc_analog/rc_analog_wpi.py
import wiringpi as wpi
PIN_CHARGE = 18
PIN_MEASURE = 23

wpi.wiringSetupGpio();
while True:
	wpi.pinMode(PIN_CHARGE, wpi.INPUT)
	wpi.pinMode(PIN_MEASURE, wpi.OUTPUT)
	wpi.digitalWrite(PIN_MEASURE, wpi.LOW)
	wpi.delay(100)
	
	wpi.pinMode(PIN_CHARGE, wpi.OUTPUT)
	wpi.pinMode(PIN_MEASURE, wpi.INPUT)
	wpi.digitalWrite(PIN_CHARGE, wpi.HIGH)
	count = 0
	
	while not(wpi.digitalRead(PIN_MEASURE)) :
		count+=1
	print(count)

File: 01gpio/05rc_analog/rc_analog_wpi.js
var wpi = require('wiring-pi');
var pin_charge = 18;
var pin_measure = 23;

wpi.wiringSetupGpio();
while(true){
	wpi.pinMode(pin_charge, wpi.INPUT);
	wpi.pinMode(pin_measure, wpi.OUTPUT);
	wpi.digitalWrite(pin_measure, wpi.LOW);
	wpi.delay(100);
	wpi.pinMode(pin_charge, wpi.OUTPUT);
	wpi.pinMode(pin_measure, wpi.INPUT);
	wpi.digitalWrite(pin_charge, wpi.HIGH);
	var count = 0;
	while(!wpi.digitalRead(pin_measure) ){
		count++;
	}
	console.log(count);
}